lib/px/utils/pxcommonutils.lua

```diff
@@ -34,11 +34,24 @@ function clone (t) -- deep-copy a table
 end
 
 function  _M.filter_config(px_config)
-    local config_copy = clone(px_config);
+    local config_copy = clone(px_config)
     -- remove
     config_copy.cookie_secret = nil
     config_copy.auth_token = nil
     return config_copy
 end
 
+function  _M.filter_headers(sensitive_headers)
```

Comment:
while you are revisiting this flow, I suggest you also clear headers from activities.
https://github.com/PerimeterX/perimeterx-nginx-plugin/blob/master/lib/px/utils/pxclient.lua#L103

It's probably better to divide this into 2 functions. One that clears the sensitive headers, and one that formats the headers in the name-value structure that's required for risk requests